package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "build": "NODE_ENV=production lerna run build",
+    "create-releases": "npx lerna version --no-push --conventional-commits --message 'chore(release): %s'",
```

Comment:
Couple changes to this script.

First, (if it wasn't before) this will now open an interactive prompt to verify the versions
<img width="842" alt="Screen Shot 2021-08-02 at 3 25 19 PM" src="https://user-images.githubusercontent.com/2503289/127931415-02bdcca7-cefe-4cef-a4aa-888c02f41c9d.png">

Second, notice the `--message 'chore(release): %s'`. Lerna will automatically fill in the version number at `%s`, so we don't have to specify the commit message.

Reply:
Does this only work when bumping the patch version (not minor/major)?